dask_expr/_concat.py

```diff
@@ -114,7 +147,14 @@ def _simplify_up(self, parent):
                 for frame, cols in zip(self._frames, columns_frame)
             ]
             return type(parent)(
```

Comment:
Seems like we are now much more likely to get empty column projections in a subset of the concatenated `DataFrame` objects. I *think* this will work just fine, but it might be good to explicitly drop items for the list in that case?

Reply:
Yep makes sense, added